Plyer_move.cpp

```diff
@@ -0,0 +1,168 @@
+/*
+���� ���: ������
+Ŭ������ 2�� ��� ����(Player, Move)
+Player Ŭ���������� ��ġ������ �߰��� ���(��ź, ���� ���)
+Move Ŭ���������� �̵��� �߰��� ���(�� �б�, ���� ������)
+Player�� �����ϴٸ� �̱��浵 ����
+*/
+
+#include <random>
+#include <conio.h>			//getch�� �ʿ�
+#include <windows.h>		//gotoxy�� �ʿ�
+#include "Plyer_move.h"
+
+using namespace std;
+
+class Player
+{
+private:
+	int x = NULL;
+	int y = NULL;
+
+	//���� ��ġ
+
+	int prev_x;
+	int prev_y;
+	//���� ��ġ
+public:
+	Player(int start_x, int start_y) : x(start_x), y(start_y), prev_x(start_x), prev_y(start_y) {}
+	//������������ġ�� ������ġ�� �ʱ�ȭ
+
+	void Move(int direction) {		//�Է¹޾� �ൿ�ϴ� �Լ�
+		prev_x = x;
+		prev_y = y;
+		//���� ��ġ�� ���
+
+		switch (direction)			//�Է¿� ���� �ൿ ����
+		{
+		case UP:
+			y--; break;
+		case DOWN:
+			y++; break;
+		case LEFT:
+			x--; break;
+		case RIGHT:
+			x++; break;
+		}
+
+		int new_x = x;		//�̵��� ��ġ�� ���
+		int new_y = y;
+
+
+		//�̵� ��ġ�� ���� ���(isWall()�� ��ȯ���� true�� ���) >>>  ���� ��ġ�� ���ư�
+		//�̵� ��ġ�� ���� �ƴ� ���(isWall()�� ��ȯ���� false�� ���) >>>  �ش� ��ġ�� �̵�	
+		if (!isWall(new_x, new_y)) {
+			x = new_x;
+			y = new_y;
+		}
+		else {
+			x = prev_x;
+			y = prev_y;
+		}
+	}
+
+	//��ǻ� �÷��̾� ��ġ�� ǥ���ϸ� ��
+	void print() {					//ȭ�� ����
+
+		gotoxy(x, y);				//�÷��̾��� ��ġ�� �ڷ� ǥ��
+		cout << "��";
```

Comment:
상단에 `#include "Text.hpp"` 를 추가하고 다음과 같이 수정하면
```suggestion
		gotoxy(x, y, "★");
```
`Text.hpp` 에 정의되어 있는 `gotoxy()` 함수를 사용할 수 있습니다.